test/goth/runner/cli/test_yagna_payment_cmd.py

```diff
@@ -1,47 +1,41 @@
 """Tests for the `runner.cli.yagna_payment_cmd` module."""
 
-import time
-
-import pytest
-
 from goth.runner.cli import Cli
+from goth.runner.cli.yagna_payment_cmd import PaymentStatus
 
 
 def test_payment_init(yagna_container):
     """Test basic usage of `payment init` command."""
 
     yagna = Cli(yagna_container).yagna
 
-    # The test fails if we call `payment init` too fast
-    time.sleep(3.0)
-    yagna.payment_init()
+    status: PaymentStatus = yagna.payment_init()
+    assert status
 
 
-@pytest.mark.skip(reason="Not sure what is the expected behaviour")
 def test_payment_init_provider_mode(yagna_container):
-    """Test `payment init -p`."""
+    """Test `payment init --receiver`."""
 
     yagna = Cli(yagna_container).yagna
 
-    yagna.payment_init(provider_mode=True)
+    status: PaymentStatus = yagna.payment_init(receiver_mode=True)
+    assert status
```

Comment:
Why do we need type annotation here? What does assert status check, if status is a PaymentStatus instance?

Reply:
See above!